### PR TITLE
Create HTML and CSS files and modify READ.me file

### PR DESCRIPTION
**Pull request by rominana**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        <div class="header-container">
+            <div class="logo">
+            <a href="#">My logo</a>
+            </div>
+        <img class="menu" src="images/Union.svg" alt="menu">
+        </div>
+    </header>
+<section class="my-name">
+    <div class="name"><h1>I'm Romina! <br>Glad to see<br> you</h1></div> 
+    <div class="greetings">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me. </div>
```
Comment (zainsadaqat) [inline index.html:21]: Please use `<p>` tag for the paragraph. Kindly replace div with p tag 👍🏻

File: index.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+</section>
+<footer>
+    <div class="connect"> LET'S CONNECT</div>
+        <div class="social-media">
+            <img class="social-media twitter" src="images/twitter.svg" alt="twitter" />
+            <img class="social-media linkedin" src="images/linkedin.svg" alt="linkedin" />
+            <img class="social-media github" src="images/github.svg" alt="github" />
+            <img class="social-media medium" src="images/medium.svg" alt="medium" />
+            <img class="social-media medium" src="images/vector.svg" alt="vector" />
+    </div>
```
Comment (zainsadaqat) [inline index.html:31]: Please use the `<ul>` `<li>` tags to display a list of items for semantic reasons 👍🏻

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Required Changes 🔁
# Hello @rominana  😄 

You're doing a great job on your project. We're proud of you. Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of Github Flow
  ✅  No linter errors

## Required Changes ♻️

- [x] Please add the description to this Pull Request like what you have done in this pull request. 👍🏻 
- [x] Social media icons are overlapping the paragraph. Kindly fix this 
Screen Shot 2022-05-26 at 7 25 37 PM

- [x] Please remove the  blank.yml file inside of .github/workflows folder as it's not needed

_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rominana** [comment]
All the required changes have been made, as also the modifications on the HTML file. Thanks, @zainsadaqat for your feedback :smiley:

File: index.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    <div class="name"><h1>I'm Romina! <br>Glad to see<br> you</h1></div> 
+    <div class="greetings"><p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p></div>
+    <div class="connect"> LET'S CONNECT</div>
+    <div class="social-media">
+        <ul>
+            <li><img class="social-media twitter" src="images/twitter.svg" alt="twitter" /></li>  
+            <li><img class="social-media linkedin" src="images/linkedin.svg" alt="linkedin"/></li>
+            <li><img class="social-media github" src="images/github.svg" alt="github" /></li>
+            <li> <img class="social-media medium" src="images/medium.svg" alt="medium" /></li>
+            <li><img class="social-media medium" src="images/vector.svg" alt="vector" /></li>
```
Comment (smunozmo) [inline index.html:29]: - [OPTIONAL] I strongly recommend using `a` tags inside the `li` tags to make the icons clickable as this is a good way to redirect the user to the links.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+<title>Rominas Microverse Student Project 1</title>
+<link rel="stylesheet" href="style.css" />
+</head>
+<body>
+    <header class="header">
+        <div class="header-container">
+            <div class="logo">
+            <a href="#">My logo</a>
+            </div>
+        <img class="menu" src="images/Union.svg" alt="menu">
```
Comment (smunozmo) [inline index.html:16]: - [OPTIONAL] I would suggest using a `nav` tag for your menu, as this will be used for navigation.

**Review by smunozmo — APPROVED** [review]
Hi @rominana, 👋 

## Highlights 👏 

- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

### APPROVED! 💯  💯  💯 




Cheers and Happy coding! 🤓 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rominana** [comment]
Thank you so much for the feedback @smunozmo! :smile: I'll definitely do those changes.

**Pull request merged**